types/node/index.d.ts

```diff
@@ -2607,6 +2634,7 @@ declare module "dns" {
     // NOTE: This namespace provides design-time support for util.promisify. Exported members do not exist at runtime.
     export namespace resolve {
         export function __promisify__(hostname: string, rrtype?: "A" | "AAAA" | "CNAME" | "NS" | "PTR"): Promise<string[]>;
```

Comment:
This is not for this PR, but I noticed that `__promisify__(hostname: string, rrtype?: "A" | ...)` prevents `__promisify__(hostname: string, rrtype?: string)` from ever matching a call like `__promisify__("host")`. I'm pretty sure that `rrtype` should not be optional in the first overload.